### DISCUSSION TITLE
Docker image on my P1467

### DISCUSSION THREAD

**Discussion by Ajiro3** [AxisCommunications/acap-computer-vision-sdk-examples#146] [Q&A] [answered]
Dear sir,
I downloaded "object-detection" from https://github.com/AxisCommunications/acap3-examples.
I successfully built the docker image of object-detection with my tflite model data and uploaded to my P1467.
I invoked my docker image from the browser and I can see its execution log with the browser shown in the attached screen shot.
Here are some questions.
Q1) Where is my docker image stored on my P1467? Please let me know its folder.
Q2) Where is the log file stored on my P1467?
Q3) Where is the inference server docker image stored?
Q4) How can I invoke the object-detection docker image from an ubuntu terminal?

**Comment by Corallo** [comment, marked as answer]
Hello @Ajiro3 

I think you are mixing up the ACAP3-SDK and the Computer Vision SDK.

In case of the [**ACAP3-SDK** ](https://github.com/AxisCommunications/acap3-examples) you produce an eap file, which is just a zip of your binary. 
When you install the application it will be located in `/mnt/flash/usr/local/packages/`. 
The log of the app is mixed with the system log, you can run `journalctl -u app_name` to filter the log of your app.
In this case, docker is involved only to build the .eap file. Not to deploy the app on the device.

In case you are using the [**Computer vision SDK**](https://github.com/AxisCommunications/acap-computer-vision-sdk-examples), you are going to use the inference server.
It should probably end up in `/var/spool/storage/SD_DISK/dockerd` 
To run the example you should follow the instructions in the readme of the [example](https://github.com/AxisCommunications/acap-computer-vision-sdk-examples/tree/main/object-detector-python)

If your questions is related to ACAP3 it would be better to move the discussion to the [ACAP3-examples repository.](https://github.com/AxisCommunications/acap3-examples)
